src/common/string-builder.ts

```diff
@@ -54,7 +54,16 @@ class StringBuilder {
     }
 
     makeParams(params?: object) {
-        if(this.config.params && params) this.printQueue.push(JSON.stringify(params));
+        if(this.config.params && params) {
+            let str: string = '';
```

Comment:
It's better to leave this part to type inference.

```suggestion
            let str = '';
```